### ISSUE TITLE
CONDA_JSON=True conda info is no longer returning json

### ISSUE THREAD

**Issue by wrighting** [conda/conda#12987]
### What happened?

`CONDA_JSON=True conda info` used to return json, now it doesn't

See also [https://github.com/conda-forge/miniforge/issues/477](https://github.com/conda-forge/miniforge/issues/477)

<details>
<summary>(conda 23.7.x) <code>CONDA_JSON=True conda info --all</code></summary>

```shell
     active environment : None
            shell level : 0
       user config file : /root/.condarc
 populated config files : /root/conda/.condarc
                          /root/.condarc
          conda version : 23.7.2
    conda-build version : not installed
         python version : 3.9.16.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.31=0
                          __linux=4.4.0=0
                          __unix=0=0
       base environment : /root/conda  (writable)
      conda av data dir : /root/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /root/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /root/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/23.7.2 requests/2.28.2 CPython/3.9.16 Linux/4.4.0-1052-aws debian/11 glibc/2.31
                UID:GID : 0:0
             netrc file : None
           offline mode : False
```

</details>

<details>
<summary>(conda 23.5.x) <code>CONDA_JSON=True conda info --all</code></summary>

```
{
  "GID": 1000,
  "UID": 1000,
  "active_prefix": null,
  "active_prefix_name": null,
  "av_data_dir": "/home/i35085/mambaforge/etc/conda",
  "av_metadata_url_base": null,
  "channels": [
    "https://conda.anaconda.org/conda-forge/linux-64",
    "https://conda.anaconda.org/conda-forge/noarch",
    "https://repo.anaconda.com/pkgs/main/linux-64",
    "https://repo.anaconda.com/pkgs/main/noarch",
    "https://repo.anaconda.com/pkgs/r/linux-64",
    "https://repo.anaconda.com/pkgs/r/noarch"
  ],
  "conda_build_version": "not installed",
  "conda_env_version": "23.3.1",
  "conda_location": "/home/i35085/mambaforge/lib/python3.10/site-packages/conda",
  "conda_prefix": "/home/i35085/mambaforge",
  "conda_shlvl": 0,
  "conda_version": "23.3.1",
  "config_files": [
    "/home/i35085/mambaforge/.condarc",
    "/home/i35085/.condarc"
  ],
  "default_prefix": "/home/i35085/mambaforge",
  "env_vars": {
    "CIO_TEST": "<not set>",
    "CONDA_EXE": "/home/i35085/mambaforge/bin/conda",
    "CONDA_JSON": "True",
    "CONDA_PYTHON_EXE": "/home/i35085/mambaforge/bin/python",
    "CONDA_ROOT": "/home/i35085/mambaforge",
    "CONDA_SHLVL": "0",
    "CURL_CA_BUNDLE": "<not set>",
    "LD_PRELOAD": "<not set>",
    "REQUESTS_CA_BUNDLE": "<not set>",
    "SSL_CERT_FILE": "<not set>",
    "WINDOWPATH": "2"
  },
  "virtual_pkgs": [
    [
      "__archspec",
      "1",
      "x86_64"
    ],
    [
      "__glibc",
      "2.31",
      "0"
    ],
    [
      "__linux",
      "5.15.0",
      "0"
    ],
    [
      "__unix",
      "0",
      "0"
    ]
  ]
}
```

</details>

**Comment by csetzkorn** [comment]
I have downgraded my miniconda to conda 22.11.1 - it shows this after running:

conda --version

and use an older version of metaflow 2.9.11 but still observe this behaviour. Any ideas? Pretty desperate ... thanks.

**Comment by jsgv** [comment]
I still see this issue in conda `23.7.4`, fix works when I use latest in `main` branch (`source ./dev/start`). 

Even though the fix commit was implemented a month ago, the change was not part of the latest release a week ago `23.7.4`.

**Comment by kenodegard** [comment]
@jsgv correct, this particular issue and the associated fix was not included in any of the patch releases, at this time it is expected to be included in the September release ([23.9.0](https://github.com/conda/conda/issues/13080)).
